dijet/tasks/util.py

```diff
@@ -0,0 +1,10 @@
+# coding: utf-8
+
+def linear_function(x, p):
+    return p[0] * x + p[1]
+
+def chi2(p, x, data, cov):
+    y_hat = linear_function(x=x, p=p)
+    residuals = data - y_hat
+    chi2 = residuals.T @ cov @ residuals
+    return chi2
```

Comment:
The parameter `cov` should be called `cov_inv` since it's the inverse covariance
matrix. I would also suggest renaming this to `chi2_linear` because one assumes a
line as a model function.
```suggestion
def chi2_linear(p, x, data, cov_inv):
   """
   Chi2 function to minimize assuming the model follows a linear function
   of `x` with parameters `p`, compared to `data` with inverse covariance
   matrix `cov_inv`.
   """
    y_hat = linear_function(x=x, p=p)
    residuals = data - y_hat
    chi2 = residuals.T @ cov_inv @ residuals
    return chi2
```